src/Row.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export type Props = {
+  children: (React.ReactElement | React.ReactElement[])[];
```

Comment:
You should use `ReactNode` in all places generally since it includes `ReactChild` as well as `ReactFragment` (which is a superset of `ReactNodeArray`). See source:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f0016483a4b1d1fd6fd636c47eea19e2d1614336/types/react/index.d.ts#L191-L200